### ISSUE TITLE
bashlib: Pass log level to ocrd calls and add command to log spec-conformant

### ISSUE THREAD

**Issue by kba** [OCR-D/core#366]
E.g.

```
$ocrd-olena-binarize -l WARN -I MAX -O WOLF,WOLF-IMG -m mets.xml -p '{"impl":"wolf"}'
12:32:31.776 INFO ocrd.workspace - Saving mets '/data/2019-12-04/mets.xml'
```

That should not log. Whereas `-L DEBUG` should log more verbose.

**Comment by bertsky** [comment]
Is this really something to be fixed in core? AFAICS currently simply `ocrd-olena-binarize` is ignorant of the runtime loglevel (as can already be accessed by `${ocrd__argv[log_level]}`). 

But maybe you want to offer a bashlib CLI command to do the logging, which encapsulates everything (message formatting included)?

**Comment by kba** [comment]
You're right, it's more of a documentation issue, i.e. calls to `ocrd workspace` etc. in ocrd_olena should always include `-l ${ocrd__argv[log_level}`.

> But maybe you want to offer a bashlib CLI command to do the logging, which encapsulates everything (message formatting included)?

Indeed we should. I'm surprised we don't already.

**Comment by bertsky** [comment]
> You're right, it's more of a documentation issue, i.e. calls to `ocrd workspace` etc. in ocrd_olena should always include `-l ${ocrd__argv[log_level}`.

Or, maybe I was not right at all: maybe we should support setting a loglevel override via _environment variable_ in ocrd_utils, and then export to that variable in bashlib's `ocrd__wrap`?

**Comment by bertsky** [comment]
> > But maybe you want to offer a bashlib CLI command to do the logging, which encapsulates everything (message formatting included)?
> 
> Indeed we should. I'm surprised we don't already.

In case you want to do this with bash code alone (not delegating to the Python implementation): I have a spec-conforming implementation in [ocrd-import](https://github.com/bertsky/workflow-configuration/blob/c907f022051a8d1df53011dab9aab7e907376671/ocrd-import#L36) already.

**Comment by kba** [comment]
Nice. Maybe create a little `ocrd_utils.sh` lib with logging and maybe the coordinate arithmetics function in ocrd_olena? Could be reused in different scenarios then.

**Comment by kba** [comment]
> support setting a loglevel override via environment variable in ocrd_utils

You're right being not right initially. Or something. Anyway, it's a good idea, let's do that.

**Comment by bertsky** [comment]
> Maybe create a little `ocrd_utils.sh` lib with logging and maybe the coordinate arithmetics function from ocrd_olena? Could be reused in different scenarios then.

This is what we discussed in #264 already. But the current solutions in ocrd_olena are in many ways fragile and quite task specific. I'd rather we actually delegated all these things to the Python implementation. And the way things work on shells, you'd need to wrap these as additional `ocrd bashlib` commands. Ideally, we could even remove the `xmlstarlet` dependency from ocrd_olena completely.
